utils/completion/daos.bash

```diff
@@ -38,16 +38,24 @@
 
 _daos_control_comp()
 {
-	args=("${COMP_WORDS[@]:1:$COMP_CWORD}")
+	COMPREPLY=($(GO_FLAGS_COMPLETION=1 ${COMP_WORDS[0]} "${args[@]}"))
```

Comment:
(lint) Prefer mapfile or read -a to split command output (or quote to avoid splitting). [SC2207]